cronos-play/getting-started_unreal/quick-start/broadcast-transactions.md

```diff
@@ -0,0 +1,11 @@
+# Broadcast Transactions
+
+## BroadcastEthTxAsync
+
+Broadcast signed eth tx
+
+<figure><img src="../../../.gitbook/assets/image (21).png" alt=""><figcaption></figcaption></figure>
+
+* `Out`: event delegate which is triggered after tx is broadcasted
+* `Signedtx`: signed tx as bytes
+* `Rpc`: cronos rpc server url
```

Comment:
Why we need to set RPC again? I think we've set this parameters before at Cronos configuration?

Reply:
Yes, BroadcastEthTxAsync is not a member of DefiWalletCoreActor, it is a global function, so it can not share the DefiWalletCoreActor's private data.

Reply:
Done